test/e2e/page.test.ts

```diff
@@ -57,7 +57,7 @@ describe('End to End Tests', () => {
 
     // Be sure to wait for all the promises to resolve before checking the result
     await waitFor(() => {
-      expect(select).toHaveTextContent('LOCALHOST: ws://127.0.0.1:9944');
+      expect(select).toHaveTextContent('Localhost: ws://127.0.0.1:9944');
```

Comment:
```suggestion
      expect(select).toHaveTextContent(`${NetworkType.LOCALHOST}: ws://127.0.0.1:9944`);
```

Reply:
That wouldn't work as that's the enum (and it isn't a string enum). That said, I'm ok with it being static.